skyportal/model_util.py

```diff
@@ -139,3 +138,12 @@ def create_token(ACLs, user_id, name):
     DBSession().add(t)
     DBSession().commit()
     return t.id
+
+
+def delete_token(token_id):
+    t = Token.query.get(token_id)
+    if t is not None:
+        DBSession().expire(t)
```

Comment:
What is this needed for?

Reply:
Ah good catch I think it's something left over from previous iterations of this PR and not needed any more